glean-core/ffi/src/lib.rs

```diff
@@ -128,10 +141,50 @@ where
     with_glean_mut(|glean| Ok(callback(glean)))
 }
 
+struct FdLogger {
```

Comment:
Can we move the logger to its own module?

Reply:
And we should document what an FdLogger is:

* unfiltered (filtering needs to happen on the other side)
* writes JSON-formatted logs to a file descriptor
* expectation: file descriptor is valid and points to either a file or a file descriptor controlled by the other side of FFI (so that it can read from it)